lib/src/utils/event_localizations.dart

```diff
@@ -177,30 +177,17 @@ abstract class EventLocalizations {
               ? i18n.youJoinedTheChat
               : i18n.joinedTheChat(targetName);
         }
-      } else if (newMembership == 'join') {
-        final newAvatar = event.content.tryGet<String>('avatar_url') ?? '';
-        final oldAvatar = event.prevContent?.tryGet<String>('avatar_url') ?? '';
-
-        final newDisplayname =
-            event.content.tryGet<String>('displayname') ?? '';
-        final oldDisplayname =
-            event.prevContent?.tryGet<String>('displayname') ?? '';
-        final stateKey = event.stateKey;
-
-        // Has the user avatar changed?
-        if (newAvatar != oldAvatar) {
-          text = i18n.changedTheProfileAvatar(targetName);
-        }
-        // Has the user displayname changed?
-        else if (newDisplayname != oldDisplayname && stateKey != null) {
-          text = i18n.changedTheDisplaynameTo(oldDisplayname, newDisplayname);
-        }
```

Comment:
Why do you delete it? Explain it for me

Reply:
Other room member shoudn't receive the event when someone change their avatar or displayname

Reply:
Why we change it in localization, why not in somewhere have the logic to filter event?

Reply:
okay remove it